packages/components/socialFeed/Map/Map.web.tsx

```diff
@@ -205,8 +206,9 @@ export const Map: FC<MapProps> = ({
         center={
           consultedPostLocation || creatingPostLocation || DEFAULT_MAP_POSITION
         }
-        zoom={12}
+        zoom={!consultedPostLocation && !creatingPostLocation ? 2 : 12}
         attributionControl={false}
+        minZoom={2}
```

Comment:
To have the minimal zoom level and avoid white borders
![image](https://github.com/user-attachments/assets/dc56413a-3d41-4b30-863c-65dac16b3936)

```suggestion
        minZoom={2.15}
```

Reply:
this seem like an arbitrary value, we should be able to compute a proper value from map width and height I think

Reply:
You're right
`2.15` is ok with our layout, but the zoom could be more optimal on desktop
The goal is "Compute the smallest `zoom` and `minZoom` value to never see the white background"

Reply:
I will implement this in #1399.

Reply:
zoom and bounds are correlated, so ok